source/connect.txt

```diff
@@ -9,5 +9,147 @@ Connect to a Deployment
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+
+Ensure that a MongoDB deployment is running before attempting to 
+connect to it with ``mongosh``. The connection method you use depends 
+on your deployment type.
+
+Local MongoDB Instance on Default Port
+--------------------------------------
+
+You can run ``mongosh`` without any command-line options
+to connect to a MongoDB instance running on your **localhost** with 
+**default port** 27017:
+
+.. code-block:: sh
+
+   mongosh
+
+Local MongoDB Instance on a Non-Default Port
+--------------------------------------------
+
+To specify a port to connect to on localhost, use the 
+:manual:`connection string </reference/connection-string>` format. For 
+example, to connect to a MongoDB instance running on localhost with a 
+non-default port 28015:
+
+.. code-block:: sh
+
+   mongosh "localhost:28015"
+
+MongoDB Instance on a Remote Host
+---------------------------------
+
+To specify a remote host and port, use a :manual:`connection string
+</reference/connection-string>`. For example, to connect to a MongoDB
+instance running on a remote host machine on port 28015:
```

Comment:
Can we make this example Atlas specific? i.e., 

For example, to connect to a MongoDB Atlas cluster, you might use the following connection string:

Reply:
IMO this is the best example because it's the most basic, and there's a note below about connecting to Atlas instead. I worry that if we use the Atlas example as the default it confuses things conceptually a little bit. Not a strong opinion so happy to go with this if you still feel it's more common/the way of the future to use Atlas.